templates/Web/Projects/AngularDefault/.template.config/template.json

```diff
@@ -62,7 +62,7 @@
       "manualInstructions": [],
       "actionId": "CB387AC0-16D0-4E07-B41A-F1EA616A7CA9",
       "args": {
-        "dict": "{'@angular/animations': '~7.2.0', '@angular/cdk': '~7.3.7', '@angular/common': '~7.2.0', '@angular/compiler': '~7.2.0', '@angular/core': '~7.2.0', '@angular/forms': '~7.2.0', '@angular/material': '^7.3.7', '@angular/platform-browser': '~7.2.0', '@angular/platform-browser-dynamic': '~7.2.0', '@angular/router': '~7.2.0', 'bootstrap': '^4.3.1', 'core-js': '^2.5.4', 'rxjs': '~6.3.3', 'tslib': '^1.9.0', 'zone.js': '~0.8.26'}",
+        "dict": "{'@angular/animations': '~7.2.0', '@angular/cdk': '~7.3.7', '@angular/common': '~7.2.0', '@angular/compiler': '~7.2.0', '@angular/core': '~7.2.0', '@angular/forms': '~7.2.0', '@angular/material': '^7.3.7', '@angular/platform-browser': '~7.2.0', '@angular/platform-browser-dynamic': '~7.2.0', '@angular/router': '~7.2.0', '@ng-bootstrap/ng-bootstrap': '^4.2.1', 'bootstrap': '^4.3.1', 'core-js': '^2.5.4', 'rxjs': '~6.3.3', 'tslib': '^1.9.0', 'zone.js': '~0.8.26'}",
```

Comment:
Just for curiosity, how is the package: `@ng-bootstrap/ng-bootstrap': '^4.2.1'` different from normal `bootstrap` package?

Reply:
ng-bootstrap is bootstrap specifically for Angular, it is a component based styling. It makes using some Boostrap class with less code. It is highly recommended by the Angular expert and experienced user. More details can be found here. https://ng-bootstrap.github.io/#/components/alert/examples